Makefile

```diff
@@ -44,6 +44,14 @@ INTEGRATION_TARGET ?= ./test/integration/...
 
 PROJECT_DIR := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
 
+E2E_TARGET ?= ./test/e2e/...
+E2E_KIND_VERSION ?= kindest/node:v1.23.12
```

Comment:
Do we support v1.23? IIUC, since we use the CEL validation, our minimal supporting version is v1.25, right?

Reply:
Makes sense, updated to v1.25.1